drivers/sensor/tmp112/tmp112.h

```diff
@@ -44,6 +44,9 @@
 struct tmp112_data {
 	int16_t sample;
 	uint16_t config_reg;
+#ifdef CONFIG_PM_DEVICE
+	uint32_t power_state;
```

Comment:
this state is not initialized (should likely be set to active on init)

Reply:
done